precode.sh

```diff
@@ -6,38 +6,45 @@
 #   dir2
 #   dir3
 #       dir4
-
+#mkdir -p task/dir{1..3} task/dir3/dir4 хотела вот так, но не работает
+ mkdir -p task/dir1 task/dir2 task/dir3/dir4
 # изменяем текущую директорию на task
-
+ cd task
 # создаём пустой файл task/dir2/empty
+ touch dir2/empty
 
 # создаём файл task/dir2/hello.sh с таким содержанием:
 # #!/bin/bash
 # echo "$1, привет!"
+echo '#!/bin/bash
+echo "$1, привет!"' > dir2/hello.sh
 
 # устанавливаем для task/dir2/hello.sh права rwxrw-r--
-
+chmod 764 dir2/hello.sh
 # сохраняем список файлов task/dir2 в task/dir2/list.txt
-
+ls dir2 > dir2/list.txt
 # копируем содержимое каталога task/dir2 в каталог task/dir3/dir4
-
+cp -r -T dir2 dir3/dir4 
 # записываем в task/dir1/summary.txt список файлов с расширением *.txt
 # находящихся в task, включая поддиректории
-
+cd .. 
+# если не выйти на католг выше, то не знаю какой первый аргумент использовать для следующего  find
+find task -name "*.txt" > task/dir1/summary.txt
```

Comment:
Если так сделать, то выходить выше не нужно
```suggestion
find . -name "*.txt" > task/dir1/summary.txt
```

Reply:
Привет, подскажи пожалуйста, почему не работает вот такой вариант "#mkdir -p task/dir{1..3} task/dir3/dir4" на 9 строке моего кода комментарий тоже оставила. В интернете пишут, что такое должно работать. 
Это мы пишем в bash или sh? В чем принципиальная разница?

Reply:
Не знаю, что у тебя за система, но на линуксе все ок
![image](https://github.com/user-attachments/assets/5a35fe88-0a12-4cee-937b-98e8ae00696a)